src/linemixing.cc

```diff
@@ -385,10 +710,15 @@ ComplexMatrix single_species_ecs_relaxation_matrix(const AbsorptionLines& band,
   // Set the off-diagonal part of the matrix for this broadener
   switch (band.Population()) {
     case PopulationType::ByMakarovFullRelmat:
-      Makarov2020etal::relaxation_matrix_offdiagonal<param>(W.imag(), T, band, sorting, band.SpeciesMass(), species_mass);
+      Makarov2020etal::relaxation_matrix_offdiagonal(W.imag(), band, sorting, species_ecs_data, T);
       break;
+    case PopulationType::ByRovibLinearDipoleLineMixing: {
+      std::cerr << "THIS CODE IS NOT TESTED!\n";
+      LinearRovibErrorCorrectedSudden::relaxation_matrix_offdiagonal(W.imag(), band, sorting, species_ecs_data, T);
+    } break;
     default:
-      ARTS_ASSERT (false, "Bad type");
+      ARTS_ASSERT ("Bad type, we don't support band population type: ", band.Population(),
```

Comment:
Assertion is missing a proper condition. Using the string as a condition will always evaluate to true and never trigger the assertion.

Reply:
Ok, I fixed it.  I am still working on the CO2 internal solver so there's a few more things changing still.  But the O2 code is good now.